src/templates/finops-hub/modules/vnet.bicep

```diff
@@ -59,6 +59,15 @@ var subnets = [
       ]
     }
   }
+  {
+    name: 'dataExplorer-subnet'
```

Comment:
I can't leave a comment on line 27, but shouldn't that be "storage-subnet"? The storage account is for staging and isn't the hub instance. If we're calling out "dataExplorer" here, I think we should be explicit about "storage" in the other one.